src/main/java/com/hivemq/client/mqtt/mqtt5/message/auth/Mqtt5AuthBuilder.java

```diff
@@ -37,7 +37,7 @@
 public interface Mqtt5AuthBuilder extends Mqtt5EnhancedAuthBuilder {
 
     @Override
-    @NotNull Mqtt5AuthBuilder data(@Nullable byte[] data);
+    @NotNull Mqtt5AuthBuilder data(byte @Nullable [] data);
```

Comment:
How can the byte in the array be null?

Reply:
These changes were made as the new IntelliJ version properly differentiates between array reference and array elements for nullability inspection.
The ild version results in a warning, as an NotNull/Nullable annotation is missing for the array reference.